iFixit/react-commerce
# Improve separation between server and client code

This is the first of a series of PRs that aim at improving overall site performance (#1156). Here we just refactor a little bit the code so that Next.js knows what code belongs to `getServerSideProps` and what code belongs to the page component. There are more things to do than these changes, but I'm trying to constraint the size of the PR so we can hopefully review it faster.

## QA

This change are basically splitting code into different files. To QA this I'd just perform a smoke test on product list pages and product pages to verify that these pages are still working.
